### PR TITLE
Update README.md with Detailed Tool Description and Images

### PR DESCRIPTION
**Pull request by mrT23**
Type of PR:
**Documentation**

___
PR Description:
**This PR updates the README.md file of the 'pr-agent' project. The changes include a more detailed description of the tool's functionalities and the addition of images to illustrate its features.**

___
PR Main Files Walkthrough:
-`README.md`: The description of the 'pr-agent' tool has been expanded to include more detailed information about its functionalities such as Auto-Description, PR Review, Question Answering, and Code Suggestion. Additionally, images have been added to visually represent some of these features. The table of contents has also been updated to reflect these changes.

### CODE REVIEW

**Comment by okotek** [comment]
/ask write a poem about this PR

File: README.md [hunk 1]
```diff
@@ -9,10 +9,23 @@
...
+**PR Review**: Feedback about the PR main theme, type, relevant tests, security issues, focused, and various suggestions for the PR content.
+\
+**Question Answering**: Answering free-text questions about the PR.
+\
+**Code Suggestion**: Committable code suggestions for improving the PR.
+
+<p float="left">
+  <img src="/pics/pr_questions.png" width="400" />
+  <img src="/pics/pr_auto_description.png" width="400" />
+</p>
```
[the remark this answers is not in the document]
Comment (okotek) [reply README.md]: that's actually a good suggestion

**Review by hussam789 — APPROVED** [review]

**Comment by okotek** [comment]
@CodiumAI-Agent

**Comment by okotek** [comment]
@CodiumAI-Agent

**Comment by okotek** [comment]
@CodiumAI-Agent /review

**Comment by okotek** [comment]
@CodiumAI-Agent /describe

**Comment by okotek** [comment]
@CodiumAI-Agent /improve

**Comment by okotek** [comment]
@CodiumAI-Agent /ask write a poem about this PR

**Comment by okotek** [comment]
@CodiumAI-Agent /improve

**Pull request merged**